Src/Core/CaseStudy.Core/Models/DealerPages.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace CaseStudy.Core.Models
+{
+    public class DealerPages
+    {
+        public int Id {  get; set; }
```

Comment:
why no attributes here?

Reply:
As Id is automatically considered as primary while creating table, I havent mentioned explicitly